### DISCUSSION TITLE
Cloudflare Pages Template has Outdated Wrangler (3.8.0)

### DISCUSSION THREAD

**Discussion by joaoguidev** [remix-run/remix#8630] [Q&A]
Hello.

I'm using Cloudflare Pages template of Remix( v 2.5.1 currently) (`npx create-remix@latest --template remix-run/remix/templates/cloudflare-pages`) and I'm getting 2 vulnerabilities flags.  I suspect it has to to to the fact the Wrangler been used is version 3.8.0 and the current version is 3.25.0.

**Is there a reason for Remix to use this specific version of Wrangler or this should  have a version bump on next versions of Remix?**

![image](https://github.com/remix-run/remix/assets/63625334/cf9ec5d1-09d4-4b2b-b4a4-b7ffbcce8d13)

**Comment by sergiodxa** [comment]
wranglers is a dependency in your package.json, you can upgrade it yourself
